geth/rpc/client.go

```diff
@@ -2,14 +2,21 @@ package rpc
 
 import (
 	"context"
+	"encoding/json"
+	"errors"
 	"fmt"
+	"reflect"
+	"sync"
 
 	"github.com/ethereum/go-ethereum/node"
 	"github.com/status-im/status-go/geth/params"
 
 	gethrpc "github.com/ethereum/go-ethereum/rpc"
 )
 
+// Handler defines handler for RPC methods.
+type Handler func(context.Context, ...interface{}) (interface{}, error)
```

Comment:
Actually, usage of this package is `rpc.Handler`, and `go lint` suggested to rename it from `RPCHandler` to `Handler` exactly because of this :) `rpc.RPCHandler` is a bit clunky.